frontend/.env.development

```diff
@@ -1,5 +1,6 @@
-# VITE_NETWORK=0x5afd
-# VITE_WEB3_GATEWAY=http://localhost:8545
-VITE_NETWORK=0x5aff
-VITE_WEB3_GATEWAY=https://testnet.sapphire.oasis.dev
-VITE_MESSAGE_BOX_ADDR=0x2dE080e97B0caE9825375D31f5D0eD5751fDf16D
+VITE_NETWORK=0x5afd
+VITE_WEB3_GATEWAY=http://localhost:8545
+VITE_MESSAGE_BOX_ADDR=0x5FbDB2315678afecb367f032d93F642f64180aa3
```

Comment:
Confirm this is correct/as first deployed contract and then pass it via env variable

Reply:
No, that's fine. Do they get uploaded to github actions automatically as an artifact?

Reply:
This is the correct address.